client/src/Components/Pages/Questionnaire/PersonalDetails1/index.js

```diff
@@ -4,45 +4,75 @@ import Select from 'react-select';
 import './style.css'
 import '../style.css'
 
-const options = [
+const options = [                     
 
-    { value: 'Math', label: 'Math' },
+    { value: 'Art', label: 'Art' },
+    { value: 'Maths', label: 'Maths' },
     { value: 'Science', label: 'Science' },
-    { value: 'English', label: 'English' },
+    { value: 'English Language', label: 'English Language' },
+    { value: 'English Literature', label: 'English Literature' },
     { value: 'physics', label: 'physics' },
     { value: 'Biology', label: 'Biology' },
-    { value: 'calculus', label: 'calculus' },
-    { value: 'history', label: 'history' },
-    { value: 'Geographically', label: 'Geographically' },
-    { value: 'technology', label: 'technology' },
+    { value: 'History', label: 'History' },
+    { value: 'Geography', label: 'Geography' },
+    { value: 'Design & technology', label: 'Design & technology' },
+    { value: 'Business Studies', label: 'Business Studies' },
+    { value: 'Chemistry', label: 'Chemistry' },
+    { value: 'Classics', label: 'Classics' },
+    { value: 'Computer Science', label: 'Computer Science' },
+    { value: 'Drama', label: 'Drama' },
+    { value: 'Economics', label: 'Economics' },
+    { value: 'Business Management', label: 'Business Management' },
+    { value: 'Health and Social Care', label: 'Health and Social Care' },
+    { value: 'ICT', label: 'ICT' },
+    { value: 'Latin', label: 'Latin' },
+    { value: 'Law', label: 'Law' },
+    { value: 'Media Studies', label: 'Media Studies' },
+    { value: 'Modern Foreign Languages', label: 'Modern Foreign Languages' },
+    { value: 'Music', label: 'Music' },
+    { value: 'Physical Education', label: 'Physical Education' },
+    { value: 'Politics', label: 'Politics' },
+    { value: 'Psychology', label: 'Psychology' },
+    { value: 'Religious Studies', label: 'Religious Studies' },
+    { value: 'Philosophy', label: 'Philosophy' },
+    { value: 'Social Sciences', label: 'Social Sciences' },
+    { value: 'Statistics', label: 'Statistics' },
+    { value: 'Welsh', label: 'Welsh' },
+    { value: 'Other', label: 'Other' },
   ];
 
 class PersonalDetails1 extends Component {
 
     state = {
-        selectedOption: null, 
+        selectedOption: [], 
```

Comment:
You might need two attributes in state: one to hold the answer for the first question (what subjects are you studying?) and one to hold the answer of the second question (what are your favorite subjects?).